test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -430,6 +430,8 @@
                 "42772",
                 "42814",
                 "42815",
+                # check PR2145 introduction
```

Comment:
this can be removed or substituted for appropriate reasoning

Reply:
Added more detailed reason